gcloud-java-contrib/gcloud-java-nio/README.md

```diff
@@ -0,0 +1,174 @@
+NIO Filesystem Provider for Google Cloud Storage
+================================================
+
+Java idiomatic NIO client for [Google Cloud Storage](https://cloud.google.com/storage/).
+
+This client library allows you to easily interact with Google Cloud Storage using Java's
+standard file system API.
```

Comment:
Yes, absolutely. I've added a little something, but we'll probably want to refine it.